lib/krikri/registry.rb

```diff
@@ -0,0 +1,45 @@
+module Krikri
+
+  ##
+  # Provide a registry of defined items that can be looked up by a token
```

Comment:
This may be nitpicky as well, but does this have to be a string? IIRC, our examples of adding a new mapping to the registry use symbols and not strings. (See also [here](https://github.com/dpla/KriKri/pull/46/files#diff-b31c2deb469ce65b5f99d9b33041a7c9R56).)